tasks/main.yml

```diff
@@ -44,3 +44,11 @@
 - name: Install Java Cryptography Extension
   apt: name=oracle-java8-unlimited-jce-policy
   when: install_cryptography_extension|bool
+
+- name: Add CA certificate to Java keystore
+  java_cert:
+    cert_alias: peopledoc-ca
+    cert_path: /usr/local/share/ca-certificates/devsecops-ca.crt
+    keystore_path: /usr/lib/jvm/java-8-oracle/jre/lib/security/cacerts
+    keystore_pass: changeit
+    state: present
```

Comment:
`cert_alias`, `cert_path` and` keystore_pass` should be role parameters, for example:
```
- hosts: servers
  tasks:
  - include_role:
      name: oracle-java
    vars:
      certificates:
        - alias: peopledoc-ca
          path: /usr/local/share/ca-certificates/devsecops-ca.crt
          password: XXX
        - alias: other-ca
          path: /usr/local/share/ca-certificates/other-ca.crt
          password: YYY
```

Reply:
I'm going to add role parameters. It should look like this: 
```
vars:
  ca_certificates:
    certificates:
      - alias: peopledoc-ca
        path: /usr/local/share/ca-certificates/devsecops-ca.crt
      - ...
    password: XXX
```